_posts/jobs/2016-09-30-android.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Senior Android Engineer"
+description: We want you!
+modified:
+categories: jobs
+author: peter_krauss
+excerpt: We are looking for a Senior Android Engineer.
+tags: [Jobs, Android, Senior]
+date: 2016-09-30T10:39:55-04:00
+---
+## About the Role
+You are a Software Engineer by heart. You have clean code for breakfast sprinkled with MVVM or MVP, then you inject your dependencies while driving reactively to work. Then you integrate continuously and release it by lunchtime. In the afternoon you get up-to-date with the latest technologies and before leaving you improve the stress testing, add a espresso test or do whatever is necessary in order to have a monkey runner proof system you are proud of. You go to bed with the DoD under your pillow and dream of unit tests with a smile on your face.
```

Comment:
What do you mean by `have clean code for breakfast`? You mean reading Clean Code? If so, make it uppercase